packages/contracts/contracts/interfaces/v0.8.x/IGenArt721Minter_PBAB.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IGenArt721Minter_PBAB {
+    /**
+     * @notice Price per token in wei updated for project `_projectId` to
+     * `_pricePerTokenInWei`.
+     */
+    event PricePerTokenInWeiUpdated(
+        uint256 indexed _projectId,
+        uint256 indexed _pricePerTokenInWei
+    );
+
+    /**
+     * @notice Currency updated for project `_projectId` to symbol
+     * `_currencySymbol` and address `_currencyAddress`.
+     */
+    event ProjectCurrencyInfoUpdated(
+        uint256 indexed _projectId,
+        address indexed _currencyAddress,
+        string _currencySymbol
+    );
+
+    /// togglePurchaseToDisabled updated
+    event PurchaseToDisabledUpdated(
+        uint256 indexed _projectId,
+        bool _purchaseToDisabled
+    );
+
+    // getter function of public variable
+    function minterType() external view returns (string memory);
+
+    function genArt721CoreAddress() external returns (address);
+
+    function minterFilterAddress() external returns (address);
+
```

Comment:
I think it probably also makes sense to include the ETH-only purchase functions in the interface as well, yeah?